Fix and simplify CUAV v5 nano debug description (#647)
PX4/px4_user_guide: en/flight_controller/cuav_v5_nano.md

5 (blk) | FMU_SWCLK | +3.3V
6 (blk) | GND | GND

The product package includes a convenient debug cable that can be connected to the `DSU7` port.

> This spits out an FTDI cable for connecting the [PX4 System Console](https://dev.px4.io/master/en/debug/system_console.html) to a computer USB port, and SWD pins used for SWD/JTAG debugging.

Reviewer: ```suggestion
This splits out an FTDI cable for connecting the [PX4 System Console](https://dev.px4.io/master/en/debug/system_console.html) to a computer USB port, and SWD pins used for SWD/JTAG debugging.
```

![CUAV Debug cable](../../assets/flight_controller/cuav_v5_nano/cuav_nano_debug_cable.jpg)

> > **Warning** The SWD cable *does not split* out the 5V cable from the port.

Reviewer: ```suggestion
> **Warning** The SWD cable uses 5V as Vref but the CPU is run at 3,3V! 

  Some JTAG adapters (SEGGER J-Link) will use the Vref voltage to set the voltage on the SWD lines.   
```

> >  For direct connection to *Segger Jlink* we recommended you connect the 3.3 Volts of `DSM`/`SBUS`/`RSSI` pin 4 as Pin 1 on the debug connector (`Vtref`).

Reviewer: ```suggestion
>  For direct connection to *Segger Jlink* we recommended you use the 3.3 Volts from pin 4 of the connector marked `DSM`/`SBUS`/`RSSI` to connect to `Vtref` (Pin 1) of the debug cable. Thus providing 3.3V and *NOT* 5V to the JTAG.
```
